Weirdly enough alsa-utils now breaks locally with this error
```
aconnect.c: In function 'error_handler':
aconnect.c:45:13: error: 'err' undeclared (first use in this function)
   45 |         if (err)
      |             ^~~
aconnect.c:45:13: note: each undeclared identifier is reported only once for each function it appears in

```
but I see it passes here, meh it packaged yesterday so I will close my blind eye to this..
Build, packages and test fine on three different pc's.

--- a/srcpkgs/alsa-lib/template
+++ b/srcpkgs/alsa-lib/template
@@ -1,6 +1,6 @@
 # Template file for 'alsa-lib'
 pkgname=alsa-lib
-version=1.2.15.3
+version=1.2.16.1
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config libtool"
@@ -10,7 +10,7 @@ maintainer="cinerea0 <cinerea0@protonmail.com>"
 license="LGPL-2.1-or-later"
 homepage="https://www.alsa-project.org/wiki/Main_Page"
 distfiles="https://www.alsa-project.org/files/pub/lib/alsa-lib-${version}.tar.bz2"
-checksum=7b079d614d582cade7ab8db2364e65271d0877a37df8757ac4ac0c8970be861e
+checksum=f740db7f488255944ffd4428416ee3390a96742856916433df468c281436480e
 
 alsa-lib-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision}"

--- a/srcpkgs/alsa-ucm-conf/template
+++ b/srcpkgs/alsa-ucm-conf/template
@@ -1,13 +1,13 @@
 # Template file for 'alsa-ucm-conf'
 pkgname=alsa-ucm-conf
-version=1.2.15.3
+version=1.2.16.1
 revision=1
 short_desc="ALSA Use Case Manager topology configurations"
 maintainer="cinerea0 <cinerea0@protonmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/alsa-project/alsa-ucm-conf"
 distfiles="https://github.com/alsa-project/alsa-ucm-conf/archive/refs/tags/v${version}.tar.gz"
-checksum=90448be6875da278f4a6e36f4ea9e6f49ce3fc44c1f02e162fb3f116f181037f
+checksum=cbfa3c34add2b22375536362a42a36b1488c1ee82503117730d62524ff653aa0
 
 do_install() {
 	vdoc ucm2/README.md

--- a/srcpkgs/alsa-utils/template
+++ b/srcpkgs/alsa-utils/template
@@ -1,6 +1,6 @@
 # Template file for 'alsa-utils'
 pkgname=alsa-utils
-version=1.2.15.2
+version=1.2.16
 revision=1
 build_style=gnu-configure
 configure_args="--with-udev-rules-dir=/usr/lib/udev/rules.d --disable-alsaconf
@@ -14,7 +14,7 @@ maintainer="cinerea0 <cinerea0@protonmail.com>"
 license="GPL-2.0-only"
 homepage="https://www.alsa-project.org/wiki/Main_Page"
 distfiles="https://www.alsa-project.org/files/pub/utils/alsa-utils-${version}.tar.bz2"
-checksum=7aaaafbfb01942113ec0c31e51f705910e81079205088ca2f8f137a3869e1a3a
+checksum=092399d5e8749a1d5e188e393157521cec4b75693b60ebb79bbce728cff2232c
 
 post_install() {
 	# Install required udev rules file.

